src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newObject = {};
```

Comment:
try to give variables more meaningful names. E.g. in this case 'newRobot' or something like that